bot/exts/backend/sync/_cog.py

```diff
@@ -29,16 +29,20 @@ async def cog_load(self) -> None:
         if guild is None:
             return
 
-        log.info("Waiting for guild to be chunked to start syncers.")
-        end = datetime.datetime.now() + datetime.timedelta(minutes=30)
-        while not guild.chunked:
+        attempts = 0
+        while True:
+            attempts += 1
+            if guild.chunked:
+                log.info("Guild was found to be chunked after %d attempt(s).", attempts)
+                break
+
+            if attempts == MAX_ATTEMPTS:
+                log.info("Guild not chunked after %d attempts, calling chunk manually.", MAX_ATTEMPTS)
+                await guild.chunk()
```

Comment:
Can this fail?

Reply:
The only documented failure is if the members intent isn't enabled, looking though the call stack it doesn't raise anywhere else.

It's also a websocket action, so won't get normal http errors.

Reply:
Sounds good